CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Fix issue in the module build process used in `buf build` that would allow proto imports from
```

Comment:
- We always say "Protobuf", not "protos", "proto", etc.
- This isn't `buf build` specific, it applies to modules and all operations on them.